src/components/ScreenWrapper.js

```diff
@@ -72,6 +72,7 @@ class ScreenWrapper extends React.Component {
             }
 
             Navigation.dismissModal();
+            KeyboardShortcut.consumeBuffer();
```

Comment:
This may be obvious, but why do we have this here?

Reply:
For example, a user triggers the search page and this is not yet mounted and he immediately presses ESC to close the page. This is a fail-safe to clear the buffer.